### DISCUSSION TITLE
How do i set a shortcut for clearing all file marks in nvim tree?

### DISCUSSION THREAD

**Discussion by DMyashkov** [nvim-tree/nvim-tree.lua#2960] [Q&A] [answered]
-- Function to clear all marks in nvim-tree
				local function clear_nvim_tree_marks()
					local marks = api.marks.list()
					for _, mark in ipairs(marks) do
						api.marks.toggle(mark.absolute_path)
					end
				end

				-- Map <leader>mc to clear all marks in nvim-tree
				vim.keymap.set("n", "<leader>mc", function()
					clear_nvim_tree_marks()
				end, opts("Clear all marks in nvim-tree"))
- ChatGPT gave me this but it does not work

**Comment by alex-courtis** [comment, marked as answer]
See `:help nvim-tree-api.marks.clear()`

**Reply by DMyashkov** [reply]
Yeah that works thanks I don't know why but when i wrote /mark nothing would show up in my help menu, but now it does

**Reply by alex-courtis** [reply]
Unfortunately, only mapped keys are shown.
